api/tbcapi/tbcapi.go

```diff
@@ -27,11 +30,40 @@ var (
 type (
 	PingRequest  protocol.PingRequest
 	PingResponse protocol.PingResponse
+	BtcHeader    struct {
+		Version uint32 `json:"version"`
+
+		// hex encoded byte array
+		PrevHash string `json:"prev_hash"`
+
+		// hex encoded byte array
+		MerkleRoot string `json:"merkle_root"`
```

Comment:
@max-sanchez @joshuasing I used hex string encoding for the following fields:
* `merkle_root`
* `prev_hash`
* `bits`

the reason I do so is to match `bitcoin-cli`

Reply:
I know that strays from the docs a bit so I wanted to call it out here

Reply:
trying to match this response https://developer.bitcoin.org/reference/rpc/getblockheader.html